pkgs/by-name/sp/spacectl/package.nix

```
@@ -17,6 +17,15 @@ buildGoModule rec {
 
   vendorHash = "sha256-SYfXG6YM0Q2rCnoTM2tYvE17uBCD8yQiW/5DTCxMPWo=";
 
+  nativeBuildInputs = [installShellFiles];
```

```suggestion
  nativeBuildInputs = [ installShellFiles ];
```

To conform with `nixfmt`